.licenserc.yaml

```diff
@@ -32,5 +32,8 @@ header:
     - pkg/generic/descriptor/tree_test.go
     - pkg/generic/httppb_test/idl/echo.pb.go
     - pkg/utils/json.go
+    - pkg/protocol/bthrift/test/kitex_gen/test/k-consts.go
```

Comment:
pkg/protocol/bthrift/test/kitex_gen/** is enough

Reply:
> 

fixed